### PR TITLE
app-i18n/fcitx-anthy Add fcitx5 version of anthy plugin

### PR DESCRIPTION
**Pull request by stalkerg**
Anthy is the most popular plugin for Japanese input; without the Anthy difficult to support Japanese input for fcitx5, and the skk plugin is not enough.

### CODE REVIEW

File: app-i18n/fcitx-anthy/fcitx-anthy-5.0.14.ebuild [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+DOCS=(AUTHORS)
+
+src_configure() {
+	local mycmakeargs=(
+		-DCMAKE_INSTALL_LIBDIR="${EPREFIX}/usr/$(get_libdir)"
+	)
+	cmake_src_configure
+}
+
+pkg_postinst() {
```
Comment (liangyongxiang) [inline app-i18n/fcitx-anthy/fcitx-anthy-5.0.14.ebuild]: By using xdg instead of xdg-utils, the pkg_postinst and pkg_postrm phases can be omitted.

Comment (stalkerg) [reply app-i18n/fcitx-anthy/fcitx-anthy-5.0.14.ebuild]: I just copy-paste it from the original ebuild from the main repo. I will change it.

**Comment by st0nie** [comment]
> ## white_circle `pkgcheck` checks passed but with something can be improved!
> 
>     * pkgcheck 0.10.22
> 
>     * the corresponding _**::gentoo**_ commit: [gentoo-mirror/gentoo@26bf149](https://github.com/gentoo-mirror/gentoo/commit/26bf1493d2c2b1bc60eb245fb3750a900dc1cfe5)
> 
> 
> QA Results:
> 
> ```
> app-i18n/fcitx-anthy
>   MissingRemoteId: missing <remote-id type="github">fcitx/fcitx5-anthy</remote-id> (inferred from URI 'https://github.com/fcitx/fcitx5-anthy')
> ```
> 
> [mag QA Keywords Explanation](https://pkgcore.github.io/pkgcheck/man/pkgcheck.html#keywords)

you need to write a `metadate.xml` in the ebuild's directiory,and add remoteid,like other ebuilds in the overlay

File: app-i18n/fcitx-anthy/fcitx-anthy-5.0.14.ebuild [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Copyright 2013-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
```
Comment (Universebenzene) [inline app-i18n/fcitx-anthy/fcitx-anthy-5.0.14.ebuild]: How about updating EAPI to 8?

Comment (stalkerg) [reply app-i18n/fcitx-anthy/fcitx-anthy-5.0.14.ebuild]: Yes, I think so. I will try to do it today.

Comment (stalkerg) [reply app-i18n/fcitx-anthy/fcitx-anthy-5.0.14.ebuild]: Thanks, done.

**Comment by Universebenzene** [comment]
@github-actions[bot] `metadata.xml` is still missing as @st0nie said

**Comment by stalkerg** [comment]
@Universebenzene oops, fixed.

**Pull request merged**